src/components/SendErc20/index.tsx

```diff
@@ -0,0 +1,143 @@
+import withTransactionKit from './withTransactionKit';
+import ApprovalTransaction from './ApprovalTransaction';
+import Erc20Input from './Erc20Input';
+
+// Required Props for the SendErc20 component
+/**
+ * @typedef {Object} SendERC20Props - Props for the SendERC20Props component.
+ * @property {string} tokenAddress - Address of ERC20 token.
+ * @property {string} receiverAddress - The Ethereum address of the receiver.
+ * @property {string} value - value of Erc20 token.
+ * @property {string} onChangeValue - gets updated value from onChangeValue.
+ * @property {number} chainId - The Ethereum chain ID.
+ * @property {ethers.Wallet} provider - The Ethereum provider.
+ * @property {number} decimals - decimals for ERC20 token(using for convert number to bignumber).
+ */
+
+/**
+ * @name SendErc20
+ * @description Using SendErc20 component to we can send Erc20 token. 
+    For sending token need tokenAddress, receiverAddress and value.
+ * @param {SendERC20Props} props - The props for the component
+ * @returns {React.ReactElement} The rendered component.
+ */
+const SendErc20 = (props: SendERC20Props) => {
+  const {
+    debug,
+    onlyEstimate,
+    onSent,
+    onEstimated,
+    handleEnterPress,
+    onExecutionStatus,
+    containerClassName,
+    buttonContainerClassName,
+    buttonClassName,
+    buttonTitle,
+  } = props;
+
+  const { estimate, send } = useEtherspotTransactions();
+
+  const [isEstimate, setIsEstimate] = useState(false);
+
+  // if disableSendOnEnter is false then fired this fn.
+  const onEnterPress = () => {
+    estimateAndSend();
+    handleEnterPress && handleEnterPress();
+  };
+
+  // Execution for estimation and send process.
+  const estimateAndSend = async () => {
+    // Return if the token address, receiver address or value not valid.
+    if (!isEmpty(erc20ValidationMessage(props)) || isEstimate) return;
+
+    try {
+      setIsEstimate(true);
+      onExecutionStatus && onExecutionStatus(true);
+      /**
+       * Any transaction that is intended to be sent to the blockchain must be estimated first. 
+         The estimation function performs several checks including cost estimation and transaction validity. This method must always be called before we send, otherwise the send method will return an error.
+       */
+      let estimatedResponse;
+      try {
+        estimatedResponse = await estimate();
+        onEstimated && onEstimated(estimatedResponse);
+      } catch (e) {
+        errorLog('Estimate failed!', e, debug);
+      }
+
+      if (!isEmpty(estimatedResponse) && !onlyEstimate) {
+        /**
+         * The send function simply sends all the EtherspotBatches, 
+           which contain all EtherspotBatch and EtherspotTransaction components to the blockchain via the Etherspot platform. 
+        */
+        try {
+          const response = await send();
+          onSent && onSent(response);
+        } catch (e) {
+          errorLog('Send failed!', e, debug);
```

Comment:
For all error logs, please ensure that a link to the documentation is also logged:

```suggestion
          errorLog('Etherspot UI: Sorry there was an error whilst sending your transaction. Please visit our documentation for more information: https://etherspot.fyi', e, debug);
```

Reply:
Fixed it.